### DISCUSSION TITLE
Blocked request still available on client

### DISCUSSION THREAD

**Discussion by gpoffet** [AdguardTeam/AdGuardHome#6617] [Q&A]
Hi all,
I just installed AdGuardHome on my Synology in a docker container. I have the today last version (v0.107.43). Everything seems ok, DNS have been updated on the clients and I can see the request logs beeing filled correctly.
Now I have activated the parental control and I'm testing it. I see in the log that the test request has been blocked but the client can still opne the page, nothing is really blocked.
Any idea what can explain that ?

I have the same issue with the services. If I block any service like Youtbe nothing happen and youtube is still available on the client even if I see the request in the logs.

Thanks in advance for your help.
Gael

**Comment by ainar-g** [comment]
See [FAQ](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#doesntblock).  Additionally, make sure that:

* Your browser's own Secure DNS feature is either disabled or is pointed to your AdGuard Home.
* Your client devices' DNS settings do not include any other addresses besides the ones of your AGH.  Android devices, if I recall correctly, will automatically switch to `8.8.8.8` if an address is only set in one of the two fields but not the other.

**Comment by gpoffet** [comment]
Thanks for your answer. I finally found the issue. The DNS configuration was not applied correctly in the router.
